apps/src/lib/kits/maker/ui/SetupGuide.jsx

```diff
@@ -158,6 +187,60 @@ const downloadButtonStyle = {
   textAlign: 'center'
 };
 
+function Description(props) {
+  if (props.microbit) {
+    return (
+      <div style={style.boardDescriptions}>
+        <div style={style.microbitDescription}>
+          <h2>{applabI18n.makerSetupCircuitPlaygroundTitle()}</h2>
+          <center>
+            <a href="https://learn.adafruit.com/introducing-circuit-playground/overview">
+              <img
+                src="../assets/maker/circuit-playground-200.jpg"
+                width={200}
+                style={style.circuitPlaygroundImg}
+              />
+            </a>
+          </center>
+          <SafeMarkdown
+            markdown={applabI18n.makerSetupCircuitPlaygroundDescription()}
+          />
```

Comment:
i think we need another iteration on this component and cleaning up some repetition to make it more readable. this chunk of UI is a good example -- every description contains this same bit of code, so it should be abstracted to make the overall component cleaner.

when i said the description should be refactored into a component, this is actually the piece i was talking about 😄 it's a repetitive chunk of HTML that each description has, which is a clear indicator for code that should be turned into a React component

Reply:
Something like this?
```
function Description(props) {
  return (
    <div>
      <h2>{props.content.title}</h2>
      <center>
        <a href={props.content.href}>
          <img src={props.content.img_src} width={200} style={props.style} />
        </a>
      </center>
      <SafeMarkdown markdown={props.content.description} />
    </div>
  );
}
```

Reply:
yep, exactly! a couple of nits though:
1. we don't need `props.content` -- `props` is already an object that expresses the data for a component, so unless there's a strong reason to add a nested object, we can just pass data as props. otherwise, you end up with components like this:
```javascript
<Description content={{title: '', href: '', description: '', imgSrc: ''}} style={{width: 200}} />

// should be
<Description title="" href="" description="" imgSrc="" style={{width: 200}} />
```

2. use camelCase instead of snake_case in JavaScript

Reply:
changes implemented